README.md

```diff
@@ -49,9 +68,19 @@ public class TestPrimeUi
     assertThat(selectOne.getSelectedItem()).isEqualTo(ps4);
   }
 
-  ...
 }
 ```
 
+## Changelog
+[web-tester Changelog](web-tester/CHANGELOG.md)
```

Comment:
somehow these links are rendered on a single line in my browser ... you may need to add an extra line in between for a nice output